lib/Dancer2/Core/Route.pm

```diff
@@ -12,6 +12,14 @@ use Type::Registry;
 
 our ( $REQUEST, $RESPONSE, $RESPONDER, $WRITER, $ERROR_HANDLER );
 
+my $count = 0;
```

Comment:
I see this is added in b4707f05, but it appears unused as of 2ff2d530.  Should it be removed from b4707f05?

Reply:
Fixed!